File: Beecrowd/8-Geometria_Computacional/2158-Ajudando_o_Tio_Cláudio-5/2185.js
let i = 1;
const readline = require('readline').createInterface({
    input: process.stdin,
    output: process.stdout
});

function processInput() {
    readline.question('', (input) => {
        try {
            const [fp, fh] = input.split(' ').map(Number);
            const edge = (fp * 5 + fh * 6) / 2;
            const vertex = 2 - (fp + fh) + edge;
            console.log(`Molecula #${i}.:.`);
            console.log(`Possui ${Math.floor(vertex)} atomos e ${Math.floor(edge)} ligacoes\n`);
            i++;
            processInput();
        } catch {
            readline.close();
        }
    });
}

processInput();

File: Beecrowd/8-Geometria_Computacional/2158-Ajudando_o_Tio_Cláudio-5/2185.py
i=1
while(True):
    try:
        fp, fh = list(map(int, input().split(' ')))
        edge = (fp*5+fh*6)/2
        vertex = 2-(fp+fh)+edge
        print(f"Molecula #{i}.:.")
        print(f"Possui {int(vertex)} atomos e {int(edge)} ligacoes\n")
        i+=1

    except:
        break